velox/exec/HashBuild.cpp

```diff
@@ -46,32 +46,39 @@ BlockingReason fromStateToBlockingReason(HashBuild::State state) {
 }
 } // namespace
 
-HashBuild::HashBuild(
-    int32_t operatorId,
-    DriverCtx* driverCtx,
-    std::shared_ptr<const core::HashJoinNode> joinNode)
-    : Operator(
-          driverCtx,
-          nullptr,
-          operatorId,
-          joinNode->id(),
-          "HashBuild",
-          joinNode->canSpill(driverCtx->queryConfig())
-              ? driverCtx->makeSpillConfig(operatorId)
-              : std::nullopt),
-      joinNode_(std::move(joinNode)),
-      joinType_{joinNode_->joinType()},
-      nullAware_{joinNode_->isNullAware()},
-      needProbedFlagSpill_{needRightSideJoin(joinType_)},
-      joinBridge_(operatorCtx_->task()->getHashJoinBridgeLocked(
-          operatorCtx_->driverCtx()->splitGroupId,
-          planNodeId())),
-      keyChannelMap_(joinNode_->rightKeys().size()) {
-  VELOX_CHECK(pool()->trackUsage());
+void HashBuild::reuseHashTable(
+    exec::BaseHashTable* baseHashTable,
+    std::shared_ptr<const core::ReusedHashTableInfo> reusedHashTableInfo) {
   VELOX_CHECK_NOT_NULL(joinBridge_);
+  joinBridge_->start();
```

Comment:
@xiaoxmeng The `HashJoinBridge::start() `method must be called before the `setHashTable()` method. The existing hash join implementation also calls `HashJoinBridge::start()` in the `Task::createDriversLocked()` method.